APISamples/powershell/powershellSamples.ps1

```diff
@@ -16,10 +16,10 @@
  PARTICULAR PURPOSE.
 ===========================================================================#>
 
-$ReportPortalUri = 'http://myserver/reports/'
+$ReportPortalUri = 'http://localhost/reports/'
 
 Write-Host "Upload an item..."
-$uploadItemPath = 'C:\reports\test.rdl'
+$uploadItemPath = 'test.rdl'
```

Comment:
is the 'test.rdl' going to be included with the samples? In other words, shouldn't we also add that file to the repo?